### DISCUSSION TITLE
Alt Key And Mouse Selectively Highlight Text

### DISCUSSION THREAD

**Discussion by blackneuron** [dbeaver/dbeaver#575] [Q&A] [answered]
This is a feature request:

Can we get support for using the mouse and the Alt key to selectively highlight text?  Love dbeaver, but this is one productivity item that is missing from it.

Thank you for all the hard work!

**Comment by serge-rider** [comment, marked as answer]
Do you mean column selection mode? IF so then this feature is already there.
You can toggle it with `Alt+Shift+A`. 
I'll add it in the SQL editor main menu.

**Comment by blackneuron** [comment]
Yes I am talking about that!  Thank you!  I must of missed it when I was googleing around.

**Comment by serge-rider** [comment]
That's not surprising because this feature isn't documented in DBeaver wiki.
It is a standard Eclipse feature.
Hopefully it'll be more obvious with toggle action in the main menu.
